docs/community/improvement-proposals.md

```diff
@@ -0,0 +1,74 @@
+# Kyuubi Project Improvement Proposals (KBIP)
+
+The purpose of an KBIP is to inform and involve the user community in major improvements to the Kyuubi codebase throughout the development process, to increase the likelihood that user needs are met.
+
+KBIPs should be used for significant user-facing or cross-cutting changes, not small incremental improvements. When in doubt, if a committer thinks a change needs an KBIP, it does.
+
+## What is a KBIP?
+
+An KBIP is similar to a product requirement document commonly used in product management.
+
+An KBIP:
+
+- Is a ticket labeled “KBIP” proposing a major improvement or change to Kyuubi
+- Follows the template defined below
+- Includes discussions on the ticket about the proposal
+
+## Who?
+
+Any **community member** can help by discussing whether an KBIP is likely to meet their needs, and by proposing KBIPs.
```

Comment:
.... discussing  `whether a KBIP is likely to meet their needs and propose KBIPs.`